client/utils/assess_id_util.py

```diff
@@ -0,0 +1,58 @@
+import re
+
+def canon(assessId):
+  """
+  Takes an assessment/question's ID and canonicalizes it across iterations of
+  a course.
+
+  >>> canon(u'>>> foo\\n')
```

Comment:
Please convert these doctests into unit tests in the `tests/` directory, so that we get better visibility on code coverage/build.